gensim/similarity_learning/evaluation_scripts/mz_eval.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd

```

Comment:
Can you remove pandas at all? You can read csv with standart module https://docs.python.org/2/library/csv.html